moped-editor/src/views/staff/StaffForm.js

```diff
@@ -324,6 +325,55 @@ const StaffForm = ({
             />
           </FormControl>
         </Grid>
+        <Grid item xs={12} md={3}>
+          <FormControl variant="standard" component="fieldset">
+            <FormLabel id="roles-label">
+              Moped User Group (MUG) Member
+            </FormLabel>
+            <Controller
+              name="is_user_group_member"
+              control={control}
+              render={({ field: { name, value, onChange } }) => (
+                <RadioGroup
+                  aria-label="roles"
+                  name={name}
+                  value={value ? "true" : "false"}
+                  onChange={(e) => {
+                    const newValue = e.target.value === "true";
+                    return onChange(newValue);
+                  }}
```

Comment:
this the RFH [transform and parse](https://react-hook-form.com/advanced-usage#TransformandParse) method where you mutate the value back and forth between the form input value and the value saved in state.